src/client/builders.ts

```diff
@@ -23,6 +24,8 @@ export const roundBuilder: ResourceBuilder<Round> = (
   uniqueContributors: obj.uniqueContributors,
   amountUSD: obj.amountUSD,
   applicationsStartTime: new Date(obj.applicationsStartTime * 1000),
+  matchAmount: obj.matchAmount,
```

Comment:
Perhaps we should include a token field.

Reply:
For now we are displaying matching amount in USD, and when we need it in the round token, we can fetch the round token symbol etc. from the contract itself via wagmi.